evalml/automl/utils.py

```diff
@@ -97,11 +97,16 @@ def make_data_splitter(
             test_size=_LARGE_DATA_PERCENT_VALIDATION, shuffle=shuffle
         )
     if problem_type == ProblemTypes.REGRESSION:
-        return KFold(n_splits=n_splits, random_state=random_seed, shuffle=shuffle)
+        kfold = KFold(n_splits=n_splits, random_state=random_seed, shuffle=shuffle)
+        # can set this to true directly since k-fold requires >1 splits
+        kfold.is_cv = True
```

Comment:
This is kind of worrisome.  The KFold class is an sklearn object.  There's not really much reason for contributors or other devs to expect this attribute added to the standard sklearn object if they don't know about this code segment that modifies it.  Maybe we should consider a simple class wrapper with the same name the inherits from `KFold` and `StratifiedKFold` but defines the property as the other splitters do.  Curious what others think...